### ISSUE TITLE
About the test set of PKU dataset

### ISSUE THREAD

**Issue by shunk031** [CyberAgentAILab/RALF#6]
Hi, @UdonDa 
Thank you for publishing this remarkable implementation!

Could you tell me where I can get `test_csv_905.csv` from? I was able to receive `train_csv_9973.csv` from the author of the PKU dataset, but there is no mention of the test set.

**Comment by UdonDa** [comment]
Hi, @shunk031 
It was just our private file, I'll update the README. It is just a csv listing the filenames. The test set of the original PKU dataset does not have any annotations. 
[https://drive.google.com/file/d/19BIHOdOzVPBqf26SZY0hu1bImIYlRqVd/view?usp=sharing](https://drive.google.com/file/d/19BIHOdOzVPBqf26SZY0hu1bImIYlRqVd/view?usp=sharing
)

I note that we split the training set into training / validation / test set for our experiments as described in Sec 4.1. The modified set is uploaded at https://github.com/CyberAgentAILab/RALF/tree/main/data_splits/splits/pku.

**Comment by shunk031** [comment]
Thank you for your quick reply.

I now understand how to create the evaluation dataset, and thanks for the updated README!
